src/lib/snarky_bn382_backend/pairing_based.ml

```diff
@@ -112,7 +132,7 @@ module Keypair = struct
     in
     (set_urs_info, load)
 
-  let () = set_urs_info "/home/izzy/pickles-new/pairing-urs"
+  let () = set_urs_info "/home/pavel/urs/pairing-urs"
```

Comment:
same comment about the path

Reply:
This has been addressed in commit 3e28ef7ec16d58af0d3c8ba087f5cad92d44907d